build-packages/webpack-i18n-runtime/build-config/config.plugin.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable no-param-reassign */
+const path = require('path');
+const WebpackI18nTracker = require('./webpack-i18n-tracker');
+
+// Provide an additional loader directory from this module
+function addResolveLoader(config) {
+    if (!config.resolveLoader) {
+        config.resolveLoader = {};
+    }
+    if (!config.resolveLoader.modules) {
+        config.resolveLoader.modules = [];
+    }
```

Comment:
These two checks above, can optimized, if the values are undefined. Destruction of properties keeps reference, so I would use destruction here.

Reply:
Destructuring is used to get properties, my target here is to set a property. How exactly do you propose to use destructuring here?